build/pause/Dockerfile_windows

```diff
@@ -13,9 +13,33 @@
 # limitations under the License.
 
 ARG BASE
+
+FROM ${BASE} as windows-base
+
+# We're disabling the DiagTrack service, as we don't need it in the pause image
+# and there's no reason for it to have any CPU usage.
+# Since we're using buildx, we can't have RUN commands in a Windows stage.
+# So we're using chntpw in order to disable the service from a Linux container
+# by modifying the Windows SYSTEM file and then copying it into the final image.
+FROM --platform=linux/amd64 alpine:3.9 as prep
+
+COPY --from=windows-base /Windows/System32/config/SYSTEM /windows/
+
+RUN apk add chntpw
+RUN printf "ed ControlSet001\Services\DiagTrack\Start\n\
```

Comment:
very interesting technique :)

Reply:
We were REALLY hoping to be able to do this and still build on Linux machines.